res/css/structures/_QuickSettingsButton.pcss

```diff
@@ -36,10 +36,10 @@ limitations under the License.
         width: 32px;
         height: 32px;
         left: 0;
-        mask-image: url("$(res)/img/element-icons/settings.svg");
```

Comment:
I assume this is still used somewhere else?

Reply:
Yes!